definitions/npm/lodash_v4.x.x/flow_v0.104.x-/test_lodash-v4.x.x.js

```diff
@@ -355,7 +355,7 @@ zip([{ x: 1 }], [{ x: 2, y: 1 }])[0][2];
 /**
  * _.zipWith
  */
-zipWith(["a", "b", "c"], [1, 2, 3], (str, num) => ({ [str]: num }));
+zipWith(["a", "b", "c"], [1, 2, 3], (str, num) => ({ [string]: num }));
```

Comment:
Humm that's definitely not the same thing, here `str` is an argument, `string` is `undefined` as far as I can see.

Reply:
You are right, I looked at it too quickly and thought it was a type. I’m not sure how you want to solve this error, I only want to allow my other Pr to go through